pkg/api/meta/metatypes/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metatypes has been deprecated and removed.
+package meta // import "k8s.io/kubernetes/pkg/api/meta/metatypes"
```

Comment:
Metrics and heketi vendor pkg/api/.  Their dependency trees include metatypes, but they don't actually reference any types.  Godeps fails because godeps walks all dependency chains without actually loading the types (which aren't necessary to import).